e2etest/clean/clean_distributor_test.go

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package clean
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ssm"
+	"sort"
+	"strconv"
+	"testing"
+)
+
+func TestCleanDistributor(t *testing.T) {
+	//set up aws go sdk config
+	awsConfig := aws.Config{
+		Region: aws.String("us-west-2"),
```

Comment:
This should come from the environment.  If we set `AWS_REGION` in the workflow step's environment this line can be dropped entirely.